Documentation/security/network/proxy/envoy.rst

```diff
@@ -16,6 +16,49 @@ for the cluster. Cilium proxy is distributed within the Cilium images.
 
 For more information on the version compatibility matrix, see `Cilium Proxy documentation <https://github.com/cilium/proxy#version-compatibility-matrix>`_.
 
+***********************
+Deployment as DaemonSet
+***********************
+
+Background
+==========
+
+When Cilium L7 functionality (Ingress, Gateway API, Network Policies with L7 functionality, L7 Protocol Visibility)
+is enabled or installed in a Kubernetes cluster, the Cilium agent starts an Envoy proxy as separate process within
+the Cilium agent pod.
+
+That Envoy proxy instance becomes responsible for proxying all matching L7 requests on that node.
+As a result, L7 traffic targeted by policies depend on the availability of the Cilium agent pod.
+
+With Cilium 1.14 it's possible to deploy the Envoy proxy as independently life-cycled DaemonSet called ``cilium-envoy``
```

Comment:
I'm not sure we need the Cilium version number here, since this text will only be available starting from the docs version accompanying 1.14.

I think this was discussed elsewhere before, but I can't remember the outcome :disappointed:. @zacharysarah do you remember, or do you have an opinion, about mentioning the version number here, knowing this will only be in version 1.14+ of the docs?

Reply:
Agree that the version isn't necessary - as this should be already given by the context / version of the documentation itself.

Changed it towards mentioning it as an _alternative_.